community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -12,63 +12,66 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Install Ramble
+- name: Install Application
   hosts: localhost
   vars:
+    app_name: ${app_name}
+    profile_script: ${profile_script}
     install_dir: ${install_dir}
-    ramble_url: ${ramble_url}
-    ramble_ref: ${ramble_ref}
+    git_url: ${git_url}
+    git_ref: ${git_ref}
     chmod_mode: ${chmod_mode}
     chown_owner: ${chown_owner}
     chgrp_group: ${chgrp_group}
+    finalize_setup_script: ${finalize_setup_script}
   tasks:
+  - name: Print Application Name
+    ansible.builtin.debug:
+      msg: "Running installation for application: {{app_name}}"
+
+  - name: Add profile script for application
+    ansible.builtin.copy:
```

Comment:
Could add something like `when: profile_script != ''` but I haven't worked out the right syntax.

Reply:
Done.